### YamlMime:ManagedReference
items:
- uid: System.Xaml.XamlSchemaException
  id: XamlSchemaException
  children:
  - System.Xaml.XamlSchemaException.#ctor
  - System.Xaml.XamlSchemaException.#ctor(System.String)
  - System.Xaml.XamlSchemaException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Xaml.XamlSchemaException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: XamlSchemaException
  nameWithType: XamlSchemaException
  fullName: System.Xaml.XamlSchemaException
  type: Class
  summary: "A exceção é acionada quando um sistema de enlace ou outro sistema de representação de esquema para XAML relatórios uma exceção para o contexto do esquema."
  remarks: "O XamlSchemaException é emitida por vários casos em classes de suporte de enlace CLR que fornecem o contexto de esquema para os serviços do .NET Framework XAML (<xref:System.Xaml.XamlXmlReader>, <xref:System.Xaml.XamlObjectWriter>, ou <xref:System.Xaml.XamlServices>).</xref:System.Xaml.XamlServices> </xref:System.Xaml.XamlObjectWriter> </xref:System.Xaml.XamlXmlReader> Um XamlSchemaException normalmente é transmitida como a exceção interna para outras exceções, tal como o <xref:System.Xaml.XamlMember.IsUnknown%2A?displayProperty=fullName>-true propriedade para um <xref:System.Xaml.XamlObjectWriterException>, pelo que deve ser invulgar para detetar um XamlSchemaException diretamente, exceto se estiver a implementar um componente, tais como um escritor de objeto especializado.</xref:System.Xaml.XamlObjectWriterException> </xref:System.Xaml.XamlMember.IsUnknown%2A?displayProperty=fullName>"
  syntax:
    content: 'public class XamlSchemaException : System.Xaml.XamlException'
  inheritance:
  - System.Exception
  - System.Xaml.XamlException
  implements: []
  inheritedMembers:
  - System.Xaml.XamlException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Xaml.XamlException.LineNumber
  - System.Xaml.XamlException.LinePosition
  - System.Xaml.XamlException.Message
  platform:
  - net462
- uid: System.Xaml.XamlSchemaException.#ctor
  id: '#ctor'
  parent: System.Xaml.XamlSchemaException
  langs:
  - csharp
  name: XamlSchemaException()
  nameWithType: XamlSchemaException.XamlSchemaException()
  fullName: System.Xaml.XamlSchemaException.XamlSchemaException()
  type: Constructor
  assemblies:
  - System.Xaml
  namespace: System.Xaml
  summary: "Inicializa uma nova instância do <xref href=&quot;System.Xaml.XamlSchemaException&quot;> </xref> classe com uma mensagem fornecido pelo sistema que descreve o erro."
  syntax:
    content: public XamlSchemaException ();
    parameters: []
  overload: System.Xaml.XamlSchemaException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Xaml.XamlSchemaException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Xaml.XamlSchemaException
  langs:
  - csharp
  name: XamlSchemaException(String)
  nameWithType: XamlSchemaException.XamlSchemaException(String)
  fullName: System.Xaml.XamlSchemaException.XamlSchemaException(String)
  type: Constructor
  assemblies:
  - System.Xaml
  namespace: System.Xaml
  summary: "Inicializa uma nova instância do <xref href=&quot;System.Xaml.XamlSchemaException&quot;> </xref> classe com uma mensagem especificada que descreve o erro."
  syntax:
    content: public XamlSchemaException (string message);
    parameters:
    - id: message
      type: System.String
      description: "A mensagem que descreve a exceção. O autor da chamada deste construtor não é necessária para se certificar de que esta cadeia tem foram localizada da cultura atual do sistema."
  overload: System.Xaml.XamlSchemaException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Xaml.XamlSchemaException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.Xaml.XamlSchemaException
  langs:
  - csharp
  name: XamlSchemaException(SerializationInfo,StreamingContext)
  nameWithType: XamlSchemaException.XamlSchemaException(SerializationInfo,StreamingContext)
  fullName: System.Xaml.XamlSchemaException.XamlSchemaException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - System.Xaml
  namespace: System.Xaml
  summary: "Inicializa uma nova instância do <xref href=&quot;System.Xaml.XamlSchemaException&quot;> </xref> classe com dados serializados."
  syntax:
    content: protected XamlSchemaException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "O objeto que contém os dados de objecto serializado."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "As nível contextual das informações sobre a origem ou destino."
  overload: System.Xaml.XamlSchemaException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Xaml.XamlSchemaException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Xaml.XamlSchemaException
  langs:
  - csharp
  name: XamlSchemaException(String,Exception)
  nameWithType: XamlSchemaException.XamlSchemaException(String,Exception)
  fullName: System.Xaml.XamlSchemaException.XamlSchemaException(String,Exception)
  type: Constructor
  assemblies:
  - System.Xaml
  namespace: System.Xaml
  summary: "Inicializa uma nova instância do <xref href=&quot;System.Xaml.XamlSchemaException&quot;> </xref> classe com uma mensagem de erro especificado e uma referência a exceção interna que é a causa desta exceção."
  remarks: "Uma exceção que é acionada como um resultado direto de uma exceção anterior deve incluir uma referência a exceção anterior no <xref:System.Exception.InnerException%2A>propriedade.</xref:System.Exception.InnerException%2A> O <xref:System.Exception.InnerException%2A>propriedade devolve o mesmo valor que é transmitido para o construtor ou `null` se o <xref:System.Exception.InnerException%2A>propriedade não fornecer o valor da exceção interna para o construtor.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public XamlSchemaException (string message, Exception innerException);
    parameters:
    - id: message
      type: System.String
      description: "A mensagem que descreve a exceção. O autor da chamada deste construtor não é necessária para se certificar de que esta cadeia tem foram localizada da cultura atual do sistema."
    - id: innerException
      type: System.Exception
      description: "A exceção é a causa da exceção atual. Se o `innerException` não é o parâmetro <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>, a exceção atual num <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> bloco que processa a exceção interna."
  overload: System.Xaml.XamlSchemaException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Xaml.XamlException
  isExternal: false
  name: System.Xaml.XamlException
- uid: System.Xaml.XamlSchemaException.#ctor
  parent: System.Xaml.XamlSchemaException
  isExternal: false
  name: XamlSchemaException()
  nameWithType: XamlSchemaException.XamlSchemaException()
  fullName: System.Xaml.XamlSchemaException.XamlSchemaException()
- uid: System.Xaml.XamlSchemaException.#ctor(System.String)
  parent: System.Xaml.XamlSchemaException
  isExternal: false
  name: XamlSchemaException(String)
  nameWithType: XamlSchemaException.XamlSchemaException(String)
  fullName: System.Xaml.XamlSchemaException.XamlSchemaException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Xaml.XamlSchemaException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Xaml.XamlSchemaException
  isExternal: false
  name: XamlSchemaException(SerializationInfo,StreamingContext)
  nameWithType: XamlSchemaException.XamlSchemaException(SerializationInfo,StreamingContext)
  fullName: System.Xaml.XamlSchemaException.XamlSchemaException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.Xaml.XamlSchemaException.#ctor(System.String,System.Exception)
  parent: System.Xaml.XamlSchemaException
  isExternal: false
  name: XamlSchemaException(String,Exception)
  nameWithType: XamlSchemaException.XamlSchemaException(String,Exception)
  fullName: System.Xaml.XamlSchemaException.XamlSchemaException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Xaml.XamlSchemaException.#ctor*
  parent: System.Xaml.XamlSchemaException
  isExternal: false
  name: XamlSchemaException
  nameWithType: XamlSchemaException.XamlSchemaException
